kata-deploy/scripts/kata-deploy.sh

```diff
@@ -264,8 +293,10 @@ function cleanup_containerd() {
 function reset_runtime() {
 	kubectl label node "$NODE_NAME" katacontainers.io/kata-runtime-
 	systemctl daemon-reload
-	systemctl restart "$1"
-	systemctl restart kubelet
+	restart_cri_runtime "$1"
+	if [ "$1" != "containerd-k3s" ]; then
+		systemctl restart kubelet
```

Comment:
Stepping back, I'm not sure if we need to restart kubelet for either runtime's either.

Out of curiosity, what was the issue with restarting kubelet in the K3s case?

Reply:
Like k3s's embedded containerd, kubelet isn't installed as a systemd service.
Let me know if the restart is safe to remove. It looks like it's been there since the beginning: https://github.com/kata-containers/packaging/commit/e642e32ed50dea5f727cbbb35b7fd6368da3e870#diff-1f1e29e0ac464e6daaef21f619af1d96R26

Reply:
Thanks for clarifying. Let me do some testing on my end, but I expect just removing is appropriate.

Reply:
(NOTE: we should have CI so I don't have to test and have everyone in the world trust me!)